src/Symfony/Component/Validator/Tests/Validator/LegacyValidator2Dot5ApiTest.php

```diff
@@ -25,6 +25,8 @@ protected function setUp()
             $this->markTestSkipped('Not supported prior to PHP 5.3.9');
         }
 
+        $this->iniSet('error_reporting', -1 & ~E_USER_DEPRECATED);
```

Comment:
this looks weird to me. The legacy validator (i.e. the 2.5-BC implementation) should not trigger deprecation warnings when running it through the tests for the 2.5 API usage